.github/workflows/release.yml

```diff
@@ -30,169 +30,21 @@ jobs:
           persist-credentials: false
           fetch-depth: 0 # Used for conventional commit ranges
 
-      ## This step installs node and sets up several matchers (regex matching for Github
-      ## Annotations). See
-      ## https://github.com/actions/setup-node/blob/25316bbc1f10ac9d8798711f44914b1cf3c4e954/src/main.ts#L58-L65
-      - uses: actions/setup-node@v2.4.0
+      - uses: Workday/canvas-kit-actions/install@v1
```

Comment:
`Workday/canvas-kit-actions/install@v1` is composite action doing all caching and node install actions.